styles/components/_layout.scss

```diff
@@ -257,6 +257,11 @@
   padding: $padLg;
 }
 
+.padBg2,
+.padBgKids > * {
```

Comment:
My fault, I was unclear. I meant update the name of the class here too to padGi.

(padBg should be padLg, I forgot to change it when I did a refactor a while back)